tests/post/test_plain_post.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from nonebug import App
+
+
+@pytest.mark.asyncio
+async def test_text_to_pic(app: App):
```

Comment:
指出这个函数是为了测试当platform是rss时也会有url比较好？
```suggestion
async def test_rss_has_url(app: App):
```
之类的